Sources/SwiftKuery/Table.swift

```diff
@@ -68,8 +70,7 @@ open class Table: Buildable {
                 if column.isPrimaryKey {
                     columnsWithPrimaryKeyProperty += 1
                 }
-            }
-            else if let label = child.label, label == "tableName" {
+            } else if let label = child.label, label == "tableName", _name.isEmpty {
```

Comment:
Given name is optional this would be better as something like.....
```
else if let name = name else {
    if let label = child.label, label == "tablename" {
        _name = child.value as! String
        return
    }
}
_name = name
```

Reply:
Um, if `tableName` is the first child, then this will return before initializing any of the columns in the table.

Reply:
Yeah, this was done after a very quick look.